src/views/CollectiveView.vue

```diff
@@ -39,12 +39,32 @@ export default {
 		...mapGetters([
 			'currentCollective',
 			'loading',
+			'printLink',
 			'showing',
 		]),
 	},
 
+	mounted() {
+		window.addEventListener('keydown', this.printKeyHandler)
+	},
+
+	beforeDestroy() {
+		window.removeEventListener('keydown', this.printKeyHandler)
+	},
+
 	methods: {
 		...mapMutations(['hide']),
+
+		/**
+		 * @param {KeyboardEvent} event the keydown event
+		 */
+		printKeyHandler(event) {
+			if ((event.metaKey || event.ctrlKey) && event.key.toLowerCase() === 'p') {
```

Comment:
`ctrl+shift+p` opens a new private window in Firefox.
Let's make sure to not overwrite this as it's unrelated.